src/Assistants/DataSources/AssistantDataSource.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Invoice\Assistants\DataSources;
+
+use Invoice\Services\SettingsService;
+use Plenty\Modules\Plugin\Contracts\PluginLayoutContainerRepositoryContract;
+use Plenty\Modules\System\Contracts\WebstoreRepositoryContract;
+use Plenty\Modules\Wizard\Models\WizardData;
+use Plenty\Modules\Wizard\Services\DataSources\BaseWizardDataSource;
+
+class AssistantDataSource extends BaseWizardDataSource
+{
+    /**
+     * @var SettingsService
+     */
+    protected $settingsService;
+
+    public function __construct(
+        SettingsService $settingsService
+    )
+    {
+        $this->settingsService = $settingsService;
+    }
+
+    /**
+     * @return WizardData WizardData
+     */
+    public function findData()
+    {
+        /** @var WizardData $wizardData */
+        $wizardData = pluginApp(WizardData::class);
+        $wizardData->data = ['default' => false];
+
+        return $wizardData;
+    }
+
+    /**
+     * @return array
+     */
+    private function getEntities()
+    {
+        $data = [];
+        $pids = $this->settingsService->getInvoiceClients();
+        foreach ($pids as $pid) {
+            $settingsExist = $this->settingsService->clientSettingsExist($pid, null);
+            if ($settingsExist) {
+                $settings = $this->settingsService->getSettingsForPlentyId($pid, null);
+                $data[$pid] = [];
```

Comment:
Das macht so keinen Sinn, weil du es in der nächsten Zeile wieder überschreibst.